install/Knative-with-Minishift.md

```diff
@@ -164,8 +164,8 @@ curl -s https://raw.githubusercontent.com/knative/docs/master/install/scripts/is
 1. Run the following to install Istio:
 
    ```shell
-   kubectl apply --filename https://github.com/knative/serving/releases/download/v0.2.2/istio-crds.yaml && \
-   oc apply -f https://storage.googleapis.com/knative-releases/serving/latest/istio.yaml
+   kubectl apply --filename https://github.com/knative/serving/releases/download/v0.3.0/istio-crds.yaml && \
+   oc apply -f https://github.com/knative/serving/releases/download/v0.3.0/istio.yaml
```

Comment:
Line 168 -- when this URL was https://storage.googleapis.com/knative-releases/serving/latest/istio.yaml, the command was `oc apply -f` instead of `kubectl apply --filename`. Should I update the command to `kubectl apply --filename`? Honestly not sure why it wasn't to begin with...

Reply:
@samodell - i will check and get back to you on this, i have not tried 0.3.0 yet

Reply:
@kameshsampath FYI, I think we're close to merging in this PR.